Backend/starter_project_dotnetforce7/BlogApp2/BlogApp.API/Controllers/BlogsController.cs

```diff
@@ -0,0 +1,66 @@
+using BlogApp.Application.Features.Blogs.CQRS.Queries;
+using BlogApp.Application.Features.Blogs.DTOs;
+using BlogApp.Application.Responses;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using BlogApp.Application.Features.Blogs.CQRS.Commands;
+
+namespace BlogApp.Api.Controllers
+{
+    [Route("api/[Controller]")]
+    [ApiController]
+    public class BlogsController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public BlogsController(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
```

Comment:
It would be better if you extend the baseController implemented by Gadisa for the seek of consistency.